### ISSUE TITLE
S3 backend fails with status 255 on Elastic CI Stack for AWS

### ISSUE THREAD

**Issue by uh-toby** [buildkite-plugins/cache-buildkite-plugin#50]
Hi team,

I've tried using the s3 backend on agents running in the Elastic CI Stack for AWS using this configuration:

```
steps:
  - label: 'test caching'
    command: echo "test caching"
    plugins:
      - cache#v0.5.0:
          backend: s3
          manifest: .buildkite/pipeline.yml
          path: .buildkite
          restore: file
          save: file
```

and am getting this error: `Error: The plugin cache post-command hook exited with status 255`.

Permission to interact with the S3 bucket are granted through the EC2 instance role. I think I've configured the `BUILDKITE_PLUGIN_S3_CACHE_BUCKET` environment variable correctly: I tested it with a step:

```
steps:
  - label: 'test s3'
    command: aws s3 sync log "s3://${BUILDKITE_PLUGIN_S3_CACHE_BUCKET}/log"
```

which completed successfully.

**Comment by pzeballos** [comment]
Hey @uh-toby! Interesting... do you think you can share more information (like some more logs)?  so we can see in which part it failed. Thanks!

**Comment by uh-toby** [comment]
Hi @pzeballos here's what the plugin logged:

```
~~~ Running plugin cache post-checkout hook
[90m$[0m /var/lib/buildkite-agent/plugins/github-com-buildkite-plugins-cache-buildkite-plugin-v0-5-0/hooks/post-checkout
Cache miss up to file-level, sorry

...

~~~ Running plugin cache post-command hook
[90m$[0m /var/lib/buildkite-agent/plugins/github-com-buildkite-plugins-cache-buildkite-plugin-v0-5-0/hooks/post-command
Saving file-level cache of .buildkite



Unknown options: 

[31m🚨 Error: The plugin cache post-command hook exited with status 255[0m
^^^ +++

```

Let me know if there's anything else that would be useful.

**Comment by pzeballos** [comment]
Thanks @uh-toby! We'll check this out and let you know if we need extra info

**Comment by toote** [comment]
Sorry for the delay in getting back to you. This was an issue with version v0.5.0 of the plugin and the implementation of the verbose option which was fixed on the very next version (see #49).

The cryptic error was because the implementation passed an empty string as a parameter when the verbose option was not set which was tried to be parsed by the aws cli tool and it was not a valid option.
